README.md

```diff
@@ -1,8 +1,15 @@
 # EOC - End Of Coffee App
```

Comment:
What about semver: [semver doc](https://semver.org/), [semver article](https://zellwk.com/blog/semantic-versioning/)

Reply:
@marekrozmus In package.json we have version 
`"version": "0.0.1",`  should I add this or we don't care right now about versioning?

Reply:
I think about something like this (maybe not npm but version):
![Screenshot 2019-09-02_12-39-51-329](https://user-images.githubusercontent.com/25374390/64109096-e7338f00-cd7e-11e9-8a5d-b1693d8ef54c.png)

Reply:
I'm not convinced about adding a version to readme because version is visible in Package.json
and version badge will not update automatically due to version changes in pacakage.json

The other badges are dynamic and they change due to repo changes.

Reply:
ok